tests/tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/jenkinsfile-runner-test-framework"
+sh_unit_directory="$test_framework_directory/shunit2"
+
+version="256.0-test"
+CWP_version="1.3"
+jenkinsfile_runner_tag="jenkins-experimental/jenkinsfile-runner-test-image"
+
+downloaded_cwp_jar="to_update"
+
+. $test_framework_directory/utilities/utils.inc
+. $test_framework_directory/utilities/cwp/custom-war-packager.inc
+. $test_framework_directory/utilities/jfr/jenkinsfile-runner.inc
+
+oneTimeSetUp() {
+  downloaded_cwp_jar=$(download_cwp "$test_framework_directory" "$CWP_version")
+  downloaded_cwp_jar=$(remove_string "$downloaded_cwp_jar" "CWP downloaded as ")
```

Comment:
Maybe we could change the response from "download_cpw()" to avoid having to do this?

Reply:
I though about that, but having only `echo "$cwp_jar_file"` looks odd to me

Reply:
Changed